pkgs/applications/misc/qcad/default.nix

```
@@ -0,0 +1,92 @@
+{ boost
+, fetchFromGitHub
+, mkDerivationWith
+, muparser
+, pkgconfig
+, qmake
+, qt5
+, stdenv
+, libGLU
+}:
+
+mkDerivationWith stdenv.mkDerivation rec {
+  pname = "qcad";
+  version = "v3.24.2.1";
```

The version attribute doesn't include the `v`.
```suggestion
  version = "3.24.2.1";
```

---

(I also noticed that 3.24.3 has been released since)